lib/case_style.ex

```diff
@@ -12,13 +12,11 @@ defmodule CaseStyle do
   @type t :: %__MODULE__{tokens: CaseStyle.Tokens.t(), from_type: module}
 
   @type nimble_parsec_error ::
-          {:error, any, any, any, any, any}
-  # {:error, list, binary, map, {pos_integer, pos_integer}, pos_integer}
+          {:error, binary, binary, map, {pos_integer, pos_integer}, pos_integer}
+  @type parser_output :: {:ok, list, binary, map, {pos_integer, pos_integer}, pos_integer}
 
-  @type parser_ouput :: {:ok, list, binary, map, {pos_integer, pos_integer}, pos_integer}
-
-  # @callback parse(input :: binary) :: {:ok, CaseStyle.t()} | parser_ouput | nimble_parsec_error
-  @callback parse(input :: binary) :: parser_ouput | nimble_parsec_error
+  # @callback parse(input :: binary) :: {:ok, CaseStyle.t()} | parser_output | nimble_parsec_error
+  @callback parse(input :: binary) :: parser_output | nimble_parsec_error
   @callback might_be?(input :: binary) :: boolean
```

Comment:
might_be? the suspense is killing me :joy:

Reply:
would have been, should have been, could have been. YOU